dmutils/apiclient/data.py

```diff
@@ -197,6 +197,15 @@ def register_framework_agreement_returned(self, supplier_id, framework_slug, use
             },
         )
 
+    def find_framework_suppliers(self, framework_slug, agreement_returned=None):
+        params = {}
+        if agreement_returned is not None:
+            params['agreement_returned'] = 'true' if agreement_returned else 'false'
```

Comment:
This could default to `agreement_return=False` and then cast it to a string a couple of lines below.